commit 47d8d9a2172f827a8dde7695f415aa6f78a82d0e
Author: Florian Weimer <fweimer@redhat.com>
Date:   Wed Nov 28 07:00:48 2018 +0100

    support_quote_string: Do not use str parameter name
    
    This avoids a build failure if this identifier is used as a macro
    in a test.

diff --git a/support/support.h b/support/support.h
index 835e7173eb566355..c3ad76901e352ee7 100644
--- a/support/support.h
+++ b/support/support.h
@@ -69,10 +69,10 @@ void support_write_file_string (const char *path, const char *contents);
    the result).  */
 char *support_quote_blob (const void *blob, size_t length);
 
-/* Quote the contents of the at STR, in such a way that the result
+/* Quote the contents of the string, in such a way that the result
    string can be included in a C literal (in single/double quotes,
    without putting the quotes into the result).  */
-char *support_quote_string (const char *str);
+char *support_quote_string (const char *);
 
 /* Returns non-zero if the file descriptor is a regular file on a file
    system which supports holes (that is, seeking and writing does not
